### ISSUE TITLE
Both loss and val_loss are nan during YOLO training

### ISSUE THREAD

**Issue by Goes2021** [david8862/keras-YOLOv3-model-set#173]
Hi,

I am trying to training a ZeroCost YOLO model with my own dataset, but all I get back is 0.0000 and for the loss and val_loss: nan.  

Seen labels:	 {'type_1': 594}
Given labels:	 ['type_1']
Overlap labels:  {'type_1'}
(13, 13)
Epoch 1/30
 - 9s - loss: nan - val_loss: nan

Epoch 00001: val_loss did not improve from inf

type_1 0.0000
mAP: 0.0000
mAP did not improve from 0.
Epoch 2/30
 - 4s - loss: nan - val_loss: nan

Epoch 00002: val_loss did not improve from inf

type_1 0.0000
mAP: 0.0000
mAP did not improve from 0.

Does anyone know the problem and the solution?
Thanks!

**Comment by david8862** [comment]
what is your training config?

**Comment by Goes2021** [comment]
What exactly do you mean? I generated my images and xml files in ImageJ and just followed all the steps in the ZeroCost YOLO notebook.

**Comment by david8862** [comment]
> What exactly do you mean? I generated my images and xml files in ImageJ and just followed all the steps in the ZeroCost YOLO notebook.

I mean what `--model_type`, `--anchors_path` and other options you used when running train.py of this repo, just like mentioned in README:

```
# python train.py --model_type=yolo3_mobilenet_lite --anchors_path=configs/yolo3_anchors.txt --annotation_file=trainval.txt --classes_path=configs/voc_classes.txt --eval_online --save_eval_checkpoint
```

**Comment by Goes2021** [comment]
I am sorry i didn't make that clear before, but i am using YOLOv2. I just run the cell path to training images with the right pathes selected and after the cell with start training.
